fuel-core/src/database/coin.rs

```diff
@@ -120,4 +123,36 @@ impl Database {
                 == asset_id
         })
     }
+
+    pub fn get_coin_config(&self) -> Result<Option<Vec<CoinConfig>>, anyhow::Error> {
+        let configs = self
+            .iter_all::<Vec<u8>, Word>(COIN, None, None, None)
+            .map(|raw_coin| -> CoinConfig {
+                let coin = raw_coin.unwrap();
```

Comment:
few `unwraps` here as well - would it be better that we rather return an `error` instead of panicking here?

Reply:
I initialing had trouble unwrapping inside of that closure because I wasn't returning an error, but I agree that it is dangerous and will adjust this to properly propagate errors to the top